src/functionBased/pages/About.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const About = (props) => {
+  console.log(props);
```

Comment:
It's usually "best practice" to get rid of "debugging" code like a console.log when pushing to production